File: Selenium/Activities/selenium_Activity6.java
package selenium_session1;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class selenium_Activity6 
{
	public static void main(String[] arg)
	{
		//create a WebDriver instance
		WebDriver driver = new FirefoxDriver();
				
		//go to a web page
		driver.get("https://training-support.net/webelements/dynamic-controls");
		
		//do stuff on the page
		System.out.println("Title of the page is : " + driver.getTitle());
		
		WebElement element = driver.findElement(By.name("checkbox"));
		System.out.println("Checkbox is visible: " + element.isDisplayed());
		
		element.click();
		
		System.out.println("Checkbox is selected: " + element.isSelected());
		
		element.click();
		
		System.out.println("Checkbox is selected: " + element.isSelected());
		
		//close the browser
		driver.quit();
	}

}

File: Selenium/Activities/selenium_activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By

#create WebDriver instance
with webdriver.Firefox() as driver:

    #go to web page
    driver.get("https://training-support.net/webelements/dynamic-controls")

    #do the stuff on the page
    pageTitle = driver.title
    print("Title of the page is : " + pageTitle)

    #Find the checkbox input element.
    checkbox = driver.find_element(By.ID, "checkbox")

    #Check if the checkbox is selected and print the result.
    print("Checkbox is selected: " , checkbox.is_selected())
    
    #Click the checkbox to select it.
    checkbox.click()
    
    #Check if the checkbox is selected again and print the result.
    print("Checkbox is selected: " , checkbox.is_selected())

    #close the browser
    driver.quit()